src/nyc_trees/templates/registration/password_reset_form.html

```diff
@@ -5,9 +5,7 @@
 
 {% block content %}
 <p>
-    {% blocktrans %}
-    Forgot your password?  Enter your email address or username in the form below and we'll send you instructions for creating a new one.
-    {% endblocktrans %}
+    Forgot your password? Enter your Trees Count! username or email address and click the “Reset” button below.
```

Comment:
oh, I hadn't read the whole line. Looks weird without any quoting around `Trees Count!`